index.js

```diff
@@ -0,0 +1,71 @@
+document.addEventListener("DOMContentLoaded", () => {
+    setupBtnListeners()
+    fetchFilmsForSelectBox()
+})
+
+const setupBtnListeners = () => {
+    const submitReview = document.querySelector("#formy-mcformface")
+    submitReview.addEventListener("submit", postReview)
+}
+
+const postReview = () => {
+    let newReviewListItem = document.createElement("li")
+    let filmName = document.querySelector("h3").innerHTML
+    newReviewListItem.id = filmName
+    let list = document.querySelector("#listy-mclistface")
+    let review = document.querySelector("#text-box").value
+    let fullReview = ": " + review
+    newReviewListItem.append(fullReview)
+    list.append(newReviewListItem)
+    clearInput()
+}
+
+document.addEventListener("submit", (event) => {
+    event.preventDefault()
+}) 
+
+const fetchFilmsForSelectBox = () => {
+    const url = "https://ghibliapi.herokuapp.com/films"
+    fetch(url)
+        .then(response => response.json())
+        .then(films => {
+            let select = document.querySelector("#select-box")
+            for(film of films) {
+                select.options[select.options.length] = new Option(film.title, film.id)
+            }
+        })
+}
+
+const selectFilm = () => {
```

Comment:
This is definitely a valid way of getting info for the movie, but essentially we are making multiple fetch request when we get all the necessary data once already when we call "fetchFilmsForSelectBox" function at the beginning. Perhaps when we call "fetchFilmsForSelectBox" function, we can save the fetched data in a global variable then re-use that data when necessary instead of calling fetch everytime the user chooses different movie.